eHealthAfrica/aether
# Aet 131 prevent cascade delete

#### What does this PR do?
Prevents entities from being deleted during a cascade delete function from its parents
Prevents updates on an orphan entity (Project Schema must me specified)

#### How to test
Create an entity linked to a schema via projectschema
Delete the schema record (related projectschema is delete via cascade-delete)
Check that the entity was not deleted (projectschema should default to null)
Try updating the entity with a null projectschema
A validation error requesting you specify a project schema is returned

#### Related JIRA ticket
[AET-131](https://jira.ehealthafrica.org/browse/AET-131)
